va/va.go

```diff
@@ -948,10 +948,21 @@ func (va *ValidationAuthorityImpl) performRemoteValidation(ctx context.Context,
 				// returned error can be a nil *probs.ProblemDetails which breaks the
 				// err != nil check so do a slightly more complicated unwrap check to
 				// make sure we don't choke on that.
-				if p, ok := err.(*probs.ProblemDetails); !ok || p != nil {
+				if p, ok := err.(*probs.ProblemDetails); ok || p != nil {
+					// If the non-nil err was a non-nil problem then we can log it at an
+					// info level, it's a normal non-success validation result and the
```

Comment:
Replace this comma with a period since it separates two full sentences (comma splice).

Also, I'd like to write `ProblemDetails` in these comments instead of just "problem" since the latter can mean problem in the colloquial sense. This is particularly noticeable in the third comment:

> Otherwise, the non-nil err was *not* a problem and represents

"not a problem" seems to mean "it's fine" as opposed to "is not of type ProblemDetails"